clients/admin-ui/cypress/e2e/datasets-classify.cy.ts

```diff
@@ -211,15 +210,8 @@ describe("Datasets with Fides Classify", () => {
 
       cy.getByTestId("approve-classification-btn").should("be.enabled").click();
 
-      // The mutations should run.
+      // The mutation should run.
       cy.wait("@putDataset");
-      cy.wait("@putClassify");
-
-      // The instance should be re-fetched.
-      cy.wait("@getClassify");
-
-      // The updated status should make the button disappear.
-      cy.getByTestId("approve-classification-btn").should("not.exist");
```

Comment:
ah, perhaps we still want the button to disappear when the collection has already been approved? I guess there isn't an easy way to tell that it's been approved vs just that all of its fields have been assigned data categories

Reply:
So glad you saw that! I actually brought it up as well [last week in standup](https://ethyca.slack.com/archives/C04484SLXK7/p1673626308868609?thread_ts=1673625498.778729&cid=C04484SLXK7) and @rsilvery opened up a UX Fix ticket that is linked in there. I think there may have been some design help required 🤔

Reply:
Personally I'd love to have the button be replaced to move to the next collection (if more exist)